nile-frontend/src/index.js

```diff
@@ -3,15 +3,18 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import 'bootstrap/dist/css/bootstrap.min.css';
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root')
+
+ReactDOM.render( < React.StrictMode >
+    <
+    App / >
```

Comment:
This looks kind of strange lol

Reply:
I know. It's because of the eslint, not working always